extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -63,51 +63,38 @@ export class AmpViewerIntegration {
     this.isWebView_ = true; //delete this before submitting
```

Comment:
Good time now? I recommend `DO NOT SUBMIT` text in the comment - then the travis will make sure to remind you to remove.